src/rapaio/math/linear/dense/SVDecomposition.java

```diff
@@ -50,65 +50,161 @@
     private static final long serialVersionUID = -502574786523851631L;
```

Comment:
I see no dramatic changes here, however this decomposition is not properly tested. I would like to properly test it, before any changes, otherwise a debug on that would be painfull